src/BuildScriptGenerator/Python/PythonPlatform.cs

```diff
@@ -214,13 +218,16 @@ private void TryLogDependencies(string pythonVersion, ISourceRepo repo)
 
         private string GetVirutalEnvironmentName(BuildScriptGeneratorContext context)
         {
-            if (context.Properties == null ||
-                !context.Properties.TryGetValue(VirtualEnvironmentNamePropertyKey, out var virtualEnvName))
-            {
-                virtualEnvName = string.Empty;
-            }
-
+            string virtualEnvName = string.Empty;
+            context.Properties?.TryGetValue(VirtualEnvironmentNamePropertyKey, out virtualEnvName);
             return virtualEnvName;
         }
+
+        private bool ShouldZipVenvDir(BuildScriptGeneratorContext context)
+        {
+            // Build property takes precedence over env var
```

Comment:
you can just remove the env var, and rely only on the property

Reply:
unless it starts with `ORYX_`, which case we can tackle this later

Reply:
yep, it's `ORYX_ZIP_VIRTUALENV_DIR`